rumqttc/src/eventloop.rs

```diff
@@ -312,6 +312,16 @@ pub(crate) async fn socket_connect(
             socket.set_recv_buffer_size(recv_buffer_size).unwrap();
         }
 
+        #[cfg(all(any(target_os = "android", target_os = "fuchsia", target_os = "linux")))]
```

Comment:
```suggestion
        #[cfg(any(target_os = "android", target_os = "fuchsia", target_os = "linux"))]
```
is `all` required here?

Reply:
fair enough